### ISSUE TITLE
Limit background usage

### ISSUE THREAD

**Issue by jcgruenhage** [ligi/IPFSDroid#21]
The node should all by itself not cause that much drain. One way to fix this, would be to shut down the node automatically when the node is not used currently and it is running on battery power.

**Comment by ligi** [comment]
Thanks for the suggestion - unfortunately I will have to wait for IPFS to implement throttling upstream and exposes an API/Command for this. I don't think it is a good idea to shutdown the node automatically as this would kill some current use-cases.

**Comment by jcgruenhage** [comment]
True.. I've opened an issue over at ipfs/go-ipfs#4137, to have this fixed upstream at some point in the future.

**Comment by jcgruenhage** [comment]
There's already a response: Running ipfs with `--routing=dhtclient` should already help. Other than that, they are working on making it lighter anyway, so this should get better :)

**Comment by jcgruenhage** [comment]
@ligi there was another tip, that might help, over at the upstream issue.
